lagom10-conductr-bundle-lib/src/main/scala/com/typesafe/conductr/bundlelib/lagom/ServiceLocatorModule.scala

```diff
@@ -14,7 +14,7 @@ import play.api.inject.{ Binding, Module }
 class ServiceLocatorModule extends Module {
 
   override def bindings(environment: Environment, configuration: Configuration): Seq[Binding[_]] =
-    if (environment.mode == Mode.Prod)
+    if (Env.isRunByConductR)
```

Comment:
It should only be true when actually invoked by ConductR i.e. when a BUNDLE_ID env var is present. We use this check in many places, so if we need to tighten up the check then we should do so as necessary. The call we make here is correct though.